### DISCUSSION TITLE
ESP32 with WebUI

### DISCUSSION THREAD

**Discussion by drom89** [grblHAL/Plugin_WebUI#8] [Q&A] [answered]
Hi, does ESP32 work with WebUI ? 
In my_machine.h there is a code #define WEBUI_ENABLE       0 // Not yet available, do not change.
So i assume that it isnt working, is that correct ?

**Comment by terjeio** [comment, marked as answer]
> So i assume that it isnt working, is that correct ?

No, both v2 and v3 can be used.

_my_machine.h_ [is not used for compilation](https://github.com/grblHAL/ESP32/blob/b84bcc108d1f0c5d22d811e1bf6d1ae486899023/main/my_machine.h#L4) by anybody I am aware of so I have not bothered to keep it up to date. Edit [CMakeLists.txt](https://github.com/grblHAL/ESP32/blob/b84bcc108d1f0c5d22d811e1bf6d1ae486899023/main/CMakeLists.txt#L44) or use the [Web Builder](http://svn.io-engineering.com:8080/).

**Reply by drom89** [reply]
Ok, i will try web builder. (im still using VS to complie...) 
Im working on CNC plasma cutter and i dont want it to be connected to my notebook thru cable. 😄 
Wifi will be much safer...
